### ISSUE TITLE
Nicehash Pool Earning not shown

### ISSUE THREAD

**Issue by tcj2001** [RainbowMiner/RainbowMiner#2468]
I can see the miner hashing on Nicehash, but when  go to pool earning, I dont see Nicehash info

**Comment by whitindllc** [comment]
do you have the correct api key's in the pool setup for nicehash

**Comment by tcj2001** [comment]
yes, I can see miners hashing on nicehash, the pool earnin does not show nicehash, but pool totals shows nicehash profit,cost etc.

**Comment by NasoLiteratus** [comment]
I also have this issue with no nicehash balance showing. I have entered the API ID, Key and secret. Also organization id. The miners and earnings are all good on the nicehash side, however the wallet doesnt show on Rainbowminer.

**Comment by whitindllc** [comment]
i havent seen this work for me in over 6 months... i just gave up....

**Comment by RainbowMiner** [comment]
That Nicehash wallet is somehow unpredictable - on my side, the wallet shows up, but for some reason others have problems. If you would send your credentials to me via eMail (to info@rbminer.net, I will delete them afterwards), I will try to find out, what is going on.

**Comment by tcj2001** [comment]
My nicehash api key was not confirmed in nicehash, thats the reason it was not showing Nicehash earning.
